crates/shared/src/http_solver/model.rs

```diff
@@ -219,7 +221,14 @@ pub struct SettledBatchAuctionModel {
     pub submitter: SubmissionPreference,
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(flatten)]
+    /// TODO remove this field once all solvers conform to sending the
+    /// success_probability
     pub score: Option<Score>,
+    /// The probability of the settlement to be mined successfully.
+    /// Expected values [0..1]
+    pub success_probability: Option<f64>,
+    /// Estimated gas usage of the settlement transaction.
+    pub gas_cost: Option<U256>,
```

Comment:
This should be an amount though, right? Solvers are not in control of the gas price (the driver is), we just want to work around or gas amount simulation being too inaccurate?

Reply:
reverted to `gas_amount` after discussion.